src/Uno.UI/UI/Xaml/Controls/TextBox/TextBoxView.Android.cs

```diff
@@ -43,7 +43,7 @@ public TextBoxView(TextBox owner)
 			InitializeBinder();
 
 			base.SetSingleLine(true);
```

Comment:
I'm feeling this should be updated too.

```suggestion
			base.SetSingleLine(owner.TextWrapping == TextWrapping.NoWrap);
```